src/Apps/Auction/Components/__stories__/RegistrationForm.story.tsx

```diff
@@ -0,0 +1,27 @@
+import { SaleAuction } from "Apps/__tests__/Fixtures/Sale"
+import React from "react"
+import { Elements, StripeProvider } from "react-stripe-elements"
+import { data as sd } from "sharify"
+import { storiesOf } from "storybook/storiesOf"
+import { Section } from "Utils/Section"
+import { WrappedRegistrationForm } from "../RegistrationForm"
+
+storiesOf("Apps/Auction/Components", module).add("RegistrationForm", () => {
+  const stripe = Stripe(sd.STRIPE_PUBLISHABLE_KEY)
+
+  return (
+    <Section>
+      <StripeProvider stripe={stripe}>
+        <Elements>
+          <WrappedRegistrationForm
+            sale={SaleAuction}
+            onSubmit={(token, actions) => {
```

Comment:
This overrides the default `onSubmit behavior` of:

1. Tokenize the credit card via Stripe
1. Create a credit card via Metaphysics
1. Create a bidder via Metaphysics

Instead, it continues to tokenize the card so we can see validation errors and raises a window alert.